gatsby_fp/src/components/fieldComponents/IntellCont.js

```diff
@@ -1,112 +1,38 @@
 import React from "react"
+import SubHeading from "./intellContComponents/SubHeading.js"
 const shortid = require("shortid")
 
 export default function IntellCont({ intellContList }) {
+
+  function findExistingContypes(intellContList){
```

Comment:
Code needs to be linted. Look at the difference below
IntellCont({ intellContList }) {     vs.      findExistingContypes(intellContList){

Reply:
I ran the linter, however for this specific instance prettier only places a space before and after the parameter if it is enclosed in curly braces like in IntellCont({ intellContList }), but if there are no curly braces, it will resemble this: findExistingContypes(intellContList)